### PR TITLE
[BEAM-242] Enable and fix checkstyle in Flink runner examples

### PR DESCRIPTION
**Pull request by jbonofre**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [X] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [X] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [X] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [X] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
Enable and fix checkstyle in Flink runner examples.

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @aljoscha 
R: @mxm

**Comment by aljoscha** [comment]
Looks good to me!

File: runners/flink/examples/src/main/java/org/apache/beam/runners/flink/examples/streaming/WindowedWordCount.java [hunk 1]
```diff
@@ -111,16 +117,17 @@ public static void main(String[] args) throws IOException {
...
 
     Pipeline pipeline = Pipeline.create(options);
 
     PCollection<String> words = pipeline
         .apply("StreamingWordCount",
             Read.from(new UnboundedSocketSource<>("localhost", 9999, '\n', 3)))
         .apply(ParDo.of(new ExtractWordsFn()))
-        .apply(Window.<String>into(SlidingWindows.of(Duration.standardSeconds(options.getWindowSize()))
+        .apply(Window.<String>into(SlidingWindows.of(
+            Duration.standardSeconds(options.getWindowSize()))
```
Comment (mxm) [inline runners/flink/examples/src/main/java/org/apache/beam/runners/flink/examples/streaming/WindowedWordCount.java]: Shouldn't this be indented once more?

**Comment by mxm** [comment]
LGTM

File: runners/flink/examples/src/main/java/org/apache/beam/runners/flink/examples/streaming/WindowedWordCount.java (hunk 1 above)
Comment (jbonofre) [inline runners/flink/examples/src/main/java/org/apache/beam/runners/flink/examples/streaming/WindowedWordCount.java]: Hmmm, checkstyle didn't complain about indentation there. Let me check (during the merge). Thanks !

**Comment by jbonofre** [comment]
Thanks guys. I'm preparing the merge.

**Pull request closed without merging**